restli-server/src/main/java/com/linkedin/restli/server/ResourceDefinition.java

```diff
@@ -41,6 +41,13 @@
    */
   String getNamespace();
 
+  /**
+   * Gets the rest.li resource java class.
+   *
+   * @return java class for this rest.li resource.
+   */
+  Class<?> getResourceClass();
```

Comment:
Add a default implementation as this is changing the interface.

Also add @Override annotation on the ResourceModel class for this method.

Reply:
I thought of add @override for `ResourceModel`, then found that all other methods there didn't add @override, I will change all of them then to be consistent.

Will add default implementation to return namespace version, although this is not breaking internal, you are right that this may break open source consumers.